core/templates/create.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```

Comment:
Shouldn't need any of this code before {% extends "base.html" %} because it's all included in the base.html then 'extended' to each template with the extends base tag